tests/makefile_uefi/Makefile

```diff
@@ -0,0 +1,52 @@
+LVGL_PATH := $(CURDIR)/../..
+ARCH = x86_64
+CC = clang
+$(info LVGL_PATH is set to: $(LVGL_PATH))
+
+CSRCS  = $(shell find $(LVGL_PATH)/src/core -type f -name '*.c')
```

Comment:
Can you use [`lvgl.mk`](https://github.com/lvgl/lvgl/blob/master/lvgl.mk) instead?

If not, what about using `find` in `src` directly? It would make maintenance simpler when new subfolders are added.

Reply:
I had to change some other files for this. If lvgl.mk is used all none lvgl includes have to happen after the feature defines.